lib/region/EU/countryCodes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file List of country codes legally considered part of the EU region.
+ */
+
+var countryCodes = {
+  AT: true, // Austria
+  BE: true, // Belgium
+  BG: true, // Bulgaria
+  CH: true, // Switzerland
+  CZ: true, // Czech Republic
+  DE: true, // Germany
+  DK: true, // Denmark
+  ES: true, // Spain
+  EE: true, // Estonia
+  FI: true, // Finland
+  FR: true, // France
+  GB: true, // Great Britain / UK
+  GR: true, // Greece
+  HR: true, // Croatia
+  HU: true, // Hungary
+  IE: true, // Ireland
+  IS: true, // Iceland
+  IT: true, // Italy
+  LT: true, // Lithuania
+  LU: true, // Luxembourg
+  LV: true, // Latvia
+  NL: true, // Netherlands
+  NO: true, // Norway
+  PL: true, // Poland
+  PT: true, // Portugal
+  RO: true, // Romania
+  SE: true, // Sweden
+  SI: true, // Slovenia
+  SK: true  // Slovakia
+};
```

Comment:
So I used the `Boolean(foo)` style cast, which is pretty slow.
I went ahead and tried out the `!!` style coercion, which is apparently much faster - even faster than the lookup without the `!!`.
http://jsperf.com/jst-7/2 (Updated).

I'll go ahead and switch to that for performance reasons, though performance is hardly the main issue here. The main goal here is a function that takes in a locale and spits out a boolean. Returning an index instead would break the black box nature of this module.